cc.config.json

```diff
@@ -66,13 +52,6 @@
         "particle": {
             "modules": ["particle"],
             "dependentAssets": [
-                "d1346436-ac96-4271-b863-1f4fdead95b0",
-                "971bdb23-3ff6-43eb-b422-1c30165a3663",
-                "17debcc3-0a6b-4b8a-b00b-dc58b885581e",
-                "711ebe11-f673-4cd9-9a83-63c60ba54c5b",
-                "c0143906-9aed-447e-9436-2ae8512d1b6e",
-                "14da1725-c4c2-42b4-ab08-ee0aeb6898b3",
-                "081cab31-dccd-428e-8652-f2404cc81c47",
                 "78e0584a-4343-4727-8f37-e14e65c2a2db"
```

Comment:
"78e0584a-4343-4727-8f37-e14e65c2a2db" is "default-billboard-material.mtl", we need